en/lessons/ecto/changesets.md

```diff
@@ -22,27 +24,31 @@ iex> %Ecto.Changeset{}
 
 As you can see, it has some potentially useful fields, but they are all empty.
 
-For a changeset to be truly useful, when we create it, we need to provide a blueprint of what the data is like. `Ecto.Schema` is exactly that — a blueprint of all fields and their types. Let's look at a simple schema for a user:
+For a changeset to be truly useful, when we create it, we need to provide a blueprint of what the data is like.
+What better blueprint for our data than the schemas we've created the define our fields and types?
+
+To save us some time, let's re-use the schema we created in the previous lesson:
 
 ```elixir
-defmodule User do
+defmodule Friends.Person do
   use Ecto.Schema
 
-  schema "users" do
-    field(:name, :string)
+  schema "people" do
+    field :name, :string
+    field :age, :integer, default: 0
   end
 end
 ```
 
-To create a changeset using the `User` schema, we are going to use `Ecto.Changeset.cast/4`:
+To create a changeset using the `Friends.Person` schema, we are going to use `Ecto.Changeset.cast/4`:
 
 ```elixir
-iex> Ecto.Changeset.cast(%User{name: "Bob"}, %{}, [:name])
-#Ecto.Changeset<action: nil, changes: %{}, errors: [], data: #User<>,
+iex> Ecto.Changeset.cast(%Friends.Person{name: "Bob"}, %{}, [:name])
+#Ecto.Changeset<action: nil, changes: %{}, errors: [], data: #Friends.Person<>,
  valid?: true>
  ```
 
- The first parameter is the original data — an empty `%User{}` struct in this case. Ecto is smart enough to find the schema based on the struct itself. Second in order are the changes we want to make — just an empty map. The third parameter is what makes `cast/4` special: it is a list of fields allowed to go through, which gives us the ability to control what fields can be changed and safe-guard the rest.
+The first parameter is the original data — an empty `%User{}` struct in this case. Ecto is smart enough to find the schema based on the struct itself. Second in order are the changes we want to make — just an empty map. The third parameter is what makes `cast/4` special: it is a list of fields allowed to go through, which gives us the ability to control what fields can be changed and safe-guard the rest.
```

Comment:
The rest of the lesson refers to a User schema and struct. Should we update to `Friends.Person`?

Reply:
@SophieDeBenedetto I am in the process of doing that so these Ecto lessons will flow together.  That isn't something we've done previously but it's feedback we've heard and I think now is as good a time as any to start 😁 

Whatcha think?

Reply:
It does look like your PR here would need to be modified: https://github.com/elixirschool/elixirschool/pull/1553/.  On the upside we can eliminate all the boilerplate.  On the downside, we need to rethink some of the associations.

Reply:
Ah yea that makes sense @doomspork. So on the one hand I'm definitely down to update that lesson but I think @notactuallypagemcconnell and @gj are using the Movie app/domain model as the basis for their lesson on querying. I also think that its okay to have one or two different domain models/apps used throughout the section, as long as each section sticks to just one--whereas here in the changeset lesson we started out talking about `Friend`s and then switched to referring to `User`s. I also think there's some value in showing _some_ repetition, i.e. generating a new app and migrations, since the audience is beginners. 

Anyways I'll defer to the general consensus--what do others think?

Reply:
@SophieDeBenedetto the switch in Changesets is only a result of this PR being a WIP 😁  

The original changeset uses User and whereas basic lesson uses Friend.  I personal don't think we should be switching from Friend, to Users, to Movies in the span of 3-4 lessons, seems a bit much IMO.  

Another reason I'd rather we not switch between different data models throughout this section is that it requires each lesson have the same repetitive setup of schema, migrations, et al which is that we have successfully avoided throughout our other lessons.

Somewhat related sidenote:  Something to keep in mind is the majority of Elixir School's traffic are non-native English speakers.  Anything we can do to make concepts and ideas flow together between, the easier it is to translate or make sense of if English is not your primarily language.  Removing unnecessary verbage, duplicated content, etc all makes the job of translators easier.

Reply:
I should also add:  We can (and probably should) most certainly add additional data models to make sensible queries in the association.  I'd like to avoid a huge divergence from what folks are already putting together as they're working through the lessons.

Maybe it's as simple as the application from `Friends` to `Example` (our more traditional namespace for example code).  With that change it from go from `Friends.Person` to `Example.Person` which then allows us to do something like  `Example.Movies` with `has_many :actors, Example.Person`